oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreService.java

```diff
@@ -343,6 +344,7 @@ private void registerNodeStore() throws IOException {
             builder.setLeaseSocketTimeout(config.mongoLeaseSocketTimeout());
             builder.setMongoDB(uri, db, config.blobCacheSize());
             builder.setCollectionCompressionType(config.collectionCompressionType());
+            builder.setPerfloggerInfoMillis(config.perfLoggerInfoMillis());
```

Comment:
either this one or the one in `configureBuilder` is a duplicate. (perhaps this one is the duplicate as the other one is also used in a few similar static cases already?)

Reply:
Removed this one. I kept it in configureBuilder.